zips/zip-0227.rst

```diff
@@ -307,33 +287,79 @@ For the ``IssueBundle``:
 
-For each ``IssueAction`` in ``IssueBundle``:
+The maximum total supply of any issued Custom Asset is denoted by the constant :math:`\mathsf{MAX\_ISSUE} := 2^{64} - 1 \!`. 
 
-- check that :math:`0 < \mathtt{assetDescSize} \leq 512\!`.
-- check that :math:`\mathsf{asset\_desc}` is a string of length :math:`\mathtt{assetDescSize}` bytes.
 
-- retrieve :math:`\mathsf{AssetBase}` from the first note in the sequence and check that :math:`\mathsf{AssetBase}` is derived from the issuance validating key :math:`\mathsf{ik}` and :math:`\mathsf{asset\_desc}` as described in the `Specification: Asset Identifier`_ section.
-- check that :math:`\mathsf{issued\_assets(AssetBase).final} \neq 1` in the global state.
-- check that every note in the ``IssueAction`` contains the same :math:`\mathsf{AssetBase}` and is properly constructed as :math:`\mathsf{note} = (\mathsf{g_d}, \mathsf{pk_d}, \mathsf{v}, \text{ρ}, \mathsf{rseed}, \mathsf{AssetBase})\!`.
+Management of the Global Issuance State
+---------------------------------------
 
-If all of the above checks pass, do the following:
+The issuance state, that is, the :math:`\mathsf{issued\_assets}` map, MUST be updated by a node during the processing of any transaction that contains burn information, or an issuance bundle.
+The issuance state is chained as follows:
 
-- For each note, 
+- The issuance state for the first block in the OrchardZSA protocol is the empty map.
+- The input issuance state for the first transaction of a block is the final issuance state of the immediately preceding block.
+- The input issuance state of each subsequent transaction in the block is the output issuance state of the immediately preceding transaction.
+- The final issuance state of a block is the output issuance state of the last transaction in the block. 
 
-  - compute the note commitment as :math:`\mathsf{cm} = \mathsf{NoteCommit^{OrchardZSA}_{rcm}}(\mathsf{repr}_{\mathbb{P}}(\mathsf{g_d}), \mathsf{repr}_{\mathbb{P}}(\mathsf{pk_d}), \mathsf{v}, \text{ρ}, \text{ψ}, \mathsf{AssetBase})` as defined in the Note Structure and Commitment section of ZIP 226 [#zip-0226-notestructure]_.
-  - Add :math:`\mathsf{cm}` to the Merkle tree of note commitments.
-  - Increase the value of :math:`\mathsf{issued\_assets(AssetBase).balance}` by the value of the note, :math:`\mathsf{v}`.
+We describe the consensus rule changes that govern the management of the global issuance state in the `Specification: Consensus Rule Changes`_ section.
+We use :math:`\mathsf{issued\_assets}_{\mathsf{IN}}` and :math:`\mathsf{issued\_assets}_{\mathsf{OUT}}\!` to denote the input issuance state and output issuance state for a transaction, respectively.
```

Comment:
Since I am using the terms input and output issuance state, the {IN} and {OUT} terms feel more natural, it would be strange to use {CURR} and {PREV} (which are more block level terms)

The idea is to only work with transaction states, and the four points in lines 313-317 define how to link it to the block states.